src/raster/types.rs

```diff
@@ -103,15 +103,7 @@ impl GdalDataType {
     /// ```
     pub fn name(&self) -> String {
         let c_str = unsafe { GDALGetDataTypeName(self.gdal_ordinal()) };
```

Comment:
`c_str` is owned by `self` so we don't need to free it?

Reply:
`GDALGetDataTypeName` returns static strings.